internal/olm/operator/bundle/install.go

```diff
@@ -87,6 +109,71 @@ func (i *Install) setup(ctx context.Context) error {
 		return err
 	}
 
+	var declcfg *declarativeconfig.DeclarativeConfig
+
+	// get index image labels.
+	catalogLabels, err := registryutil.GetImageLabels(ctx, nil, i.IndexImageCatalogCreator.IndexImage, false)
+	if err != nil {
+		return fmt.Errorf("get index image labels: %v", err)
+	}
+
+	// set the field to true if FBC label is on the image or for a default index image.
+	if _, hasFBCLabel := catalogLabels[containertools.ConfigsLocationLabel]; hasFBCLabel || i.IndexImageCatalogCreator.IndexImage == registry.DefaultIndexImage {
+		i.IndexImageCatalogCreator.HasFBCLabel = true
+	}
+
+	// generate an fbc if an fbc specific label is found on the image or for a default index image.
+	if i.IndexImageCatalogCreator.HasFBCLabel {
```

Comment:
Is there a way to restructure this if statement such that we can outdent its body? I wonder if the body should be in its own function?

Reply:
Good idea. Fixed in [commit](https://github.com/operator-framework/operator-sdk/pull/5671/commits/20f18a072d3a82622896f88897ebf27253ec5284)